Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2560,7 +2560,7 @@ HighlightAPIEnabled:
 
 HyperlinkAuditingEnabled:
   type: bool
-  status: developer
+  status: embedder
```

Comment:
Why isn't this "stable" for example? I don't understand what makes this an embedder feature.